docs/source/api.rst

```diff
@@ -11,11 +12,8 @@ Fixture Functions
 .. automodule:: pytest_mock_resources.fixture.database.generic
     :members: Credentials
 
+Fixture Config
+--------------
 
 .. automodule:: pytest_mock_resources
-    :members: patch_create_engine, patch_psycopg2_connect
-
-    Patch Functions
-    ---------------
-
-    :members: patch_create_engine, patch_psycopg2_connect
```

Comment:
don't these need to still exist somewhere?

Reply:
i'm not sure what this is, i think it might have been an automatic rebase conflict thing. i'll put this back